package.json

```diff
@@ -30,6 +30,10 @@
     "@eeacms/volto-widget-temporal-coverage": "^2.0.0",
```

Comment:
@nileshgulia1 Does this PR depends on https://github.com/eea/volto-widget-temporal-coverage/pull/10?

Reply:
@avoinea Yes, its ready.